core/src/repair_service.rs

```diff
@@ -62,50 +50,36 @@ impl RepairService {
         exit: Arc<AtomicBool>,
         repair_socket: &Arc<UdpSocket>,
         cluster_info: &Arc<RwLock<ClusterInfo>>,
-        repair_slot_range: RepairSlotRange,
+        _repair_slot_range: RepairSlotRange,
     ) {
-        let mut repair_info = RepairInfo::new();
         let id = cluster_info.read().unwrap().id();
         loop {
             if exit.load(Ordering::Relaxed) {
                 break;
             }
 
-            let repairs = Self::generate_repairs(
-                blocktree,
-                MAX_REPAIR_LENGTH,
-                &mut repair_info,
-                &repair_slot_range,
-            );
+            let repairs = Self::generate_repairs(blocktree, MAX_REPAIR_LENGTH);
 
             if let Ok(repairs) = repairs {
                 let reqs: Vec<_> = repairs
                     .into_iter()
                     .filter_map(|repair_request| {
-                        let (slot, blob_index, is_highest_request) = {
-                            match repair_request {
-                                RepairType::Blob(s, i) => (s, i, false),
-                                RepairType::HighestBlob(s, i) => (s, i, true),
-                            }
-                        };
                         cluster_info
                             .read()
                             .unwrap()
-                            .window_index_request(slot, blob_index, is_highest_request)
-                            .map(|result| (result, slot, blob_index))
+                            .repair_request(&repair_request)
+                            .map(|result| (result, repair_request))
                             .ok()
                     })
                     .collect();
 
-                for ((to, req), slot, blob_index) in reqs {
+                for ((to, req), repair_request) in reqs {
                     if let Ok(local_addr) = repair_socket.local_addr() {
                         submit(
                             influxdb::Point::new("repair_service")
-                                .add_field("repair_slot", influxdb::Value::Integer(slot as i64))
-                                .to_owned()
                                 .add_field(
-                                    "repair_blob",
-                                    influxdb::Value::Integer(blob_index as i64),
+                                    "repair_request",
```

Comment:
this is gonna muck with our dashboard ;)

Reply:
Also updated the dashboard